### ISSUE TITLE
IOS 10 - All the coloured images are became white

### ISSUE THREAD

**Issue by shahaf** [cxa/UIMenuItem-CXAImageSupport#9]
Hi,
I've tested my app in iOS 10 beta and found that the menu items became white:
# Before:

![before](https://cloud.githubusercontent.com/assets/840778/16423291/2118de68-3d64-11e6-9fb5-81bc121ecbeb.png)
# And now with iOS 10 this look like the following:

![after](https://cloud.githubusercontent.com/assets/840778/16423322/3af3c528-3d64-11e6-81cf-e222572e3a68.png)

Thanks in advance

**Comment by iOSBrett** [comment]
Yep same problem, I don't think we will get a fix as this library is deprecated.  Did you find a workaround @shahaf ?

**Comment by shahaf** [comment]
I did what Jfahrenkrug offered for the solution in the MenuItemKit git.
